src/app-data/DataState.ts

```diff
@@ -63,11 +63,12 @@ export const kindsLabel: Record<MeasurementKind, string> = {
 
 export type Loader = (
   fileCollection: FileCollection,
-) => Promise<Partial<Measurements>>;
+  logger?: boolean,
+) => Promise<Measurements>;
 
 export function mergeMeasurements(
-  measurements: Partial<Measurements>,
-  newMeasurements: Partial<Measurements>,
+  measurements: Measurements,
+  newMeasurements: Measurements,
```

Comment:
Please don't revert this improvement.

Reply:
If all the loaders return `Measurements` wouldn't it need to be removed? Previously there were some returning Partial Measurements, and some the full object.

Reply:
All loaders should return `Partial<Measurements>`.

Reply:
I see. Basically all those changes were triggered by assuming that `const newMeasurements =  getEmptyMeasurement()` was correct and would help making all the loaders the same, in which case the return would be Measurements. I will try to understand better what it was intended.